src/deepsparse/loggers/function_logger.py

```diff
@@ -89,3 +90,18 @@ def log(self, identifier: str, value: Any, category: MetricCategories):
                 )
                 self._function_call_counter = 0
             self._function_call_counter += 1
+
+    def __str__(self):
+        prefix = "  "
+        text = "\n" + textwrap.indent(
+            f"target identifier: {self.target_identifier}", prefix=prefix
+        )
+        text += "\n" + textwrap.indent(
+            f"function name: {self.function.__name__}", prefix=prefix
+        )
+        text += "\n" + textwrap.indent(
+            f"frequency: {str(self.frequency)}", prefix=prefix
+        )
+        text += "\n" + textwrap.indent("target_logger: ", prefix=prefix)
+        text += "\n" + textwrap.indent(str(self.logger), prefix=2 * prefix)
```

Comment:
also wonder if there's a way we can avoid repeatedly printing the leaf loggers as this could get pretty long